src/Dfc.CourseDirectory.Database/Pttcd/StoredProcedures/RefreshFindACourseIndex.sql

```diff
@@ -0,0 +1,160 @@
+﻿CREATE PROCEDURE [Pttcd].[RefreshFindACourseIndex]
```

Comment:
does dacpac deal with the alter/drop/create thing for us?

Reply:
Thought, now that this logic has been distilled down to a stored proc, do we want to pull some or all of this logic up to the c# layer to make it more manageable and give us compile time checks, or is there some reason this needs to stay down in the sql layer? I'm thinking that given it's one provider at a time performance might be less of an issue. Linq should be able to create an efficient query for loading all the data in one hit into memory, then we can execute a merge that's parameterized on the c# side.
I think any use of stored procs has to be strongly justified given the extra maintenance overhead the incur compared to pure c#.

Reply:
> does dacpac deal with the alter/drop/create thing for us?
It does

Reply:
> Thought, now that this logic has been distilled down to a stored proc, do we want to pull some or all of this logic up to the c# layer to make it more manageable and give us compile time checks, or is there some reason this needs to stay down in the sql layer? I'm thinking that given it's one provider at a time performance might be less of an issue. Linq should be able to create an efficient query for loading all the data in one hit into memory, then we can execute a merge that's parameterized on the c# side.
I think any use of stored procs has to be strongly justified given the extra maintenance overhead the incur compared to pure c#.

We could look into that, for sure